pkg/storage/schema.go

```diff
@@ -30,6 +28,8 @@ const (
 	ReadsFromNewSchemaVersion uint = 4
 	// NewSchemaOnlyVersion Lowest version that stops dual-writes in preparation to drop old schema
 	NewSchemaOnlyVersion uint = 6
+	// MinimumSchemaVersion Lowest version of database schema current code is able to handle
+	MinimumSchemaVersion uint = 13
```

Comment:
On our BB side @ensary notes that we can't rollback once we migrate to schema version 15. Is this the same thing as needing our minimum schema version to be 15? I'm not clear on what happens if we migrate an environment to step up to 15 and then do a redeploy a week later, does this property being 13 mean on redeploy AI-API will start at 13?

Also I know this PR wasn't made to address it, but I see that we have properties around dual writes above, should these be removed now that release v0.40.0 of AI-API has taken those out?

Reply:
I should probably make this schema version 15 if we can't go backward from that

Reply:
ok so 15 fails the builds, looking into why now

Reply:
I think we need to also update the min version in main_test.go https://github.com/asecurityteam/asset-inventory-api/blob/master/integration/tests/main_test.go#L34, I didn't think about that with the PR to remove all of the old schema code. I'm surprised my PR went through, I didn't update this and tests still showed passing...

Reply:
Ill do that now

Reply:
we'll likely keep 13 here.